apps/arweave/src/ar_data_sync_sup.erl

```diff
@@ -43,12 +43,20 @@ init([]) ->
 		fun(StorageModule) ->
 			StoreID = ar_storage_module:id(StorageModule),
 			Name = list_to_atom("ar_data_sync_" ++ StoreID),
-			?CHILD_WITH_ARGS(ar_data_sync, worker, Name, [Name, StoreID])
+			?CHILD_WITH_ARGS(ar_data_sync, worker, Name, [Name, {StoreID, none}])
 		end,
 		Config#config.storage_modules
 	),
-
 	DefaultStorageModuleWorker = ?CHILD_WITH_ARGS(ar_data_sync, worker,
-		ar_data_sync_default, [ar_data_sync_default, "default"]),
-	Children = SyncWorkers ++ StorageModuleWorkers ++ [DefaultStorageModuleWorker],
+		ar_data_sync_default, [ar_data_sync_default, {"default", none}]),
+	RepackInPlaceWorkers = lists:map(
```

Comment:
Do we need to spawn ar_data_sync workers for repack_in_place storage modules? I suspect we don't want to sync new data for a storage module that's been repacked in place (right?) - but maybe ar_data_sync gets used for something else that is required while repacking in place?

Reply:
We use them to fetch chunk metadata to filter out small chunks [here](https://github.com/ArweaveTeam/arweave/blob/feature/in-place-sm-repacking/apps/arweave/src/ar_chunk_storage.erl#L847).

Reply:
Got it. Thanks!